Create ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md (#1759)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-6-11-ui-error-when-editing-smartrule-from-application-context-v-2-5430.md

---
date: ""

> title: Fix UI error when editing Smart rule from Application context

Reviewer: ```suggestion
title: Editing smart rules from the application context no longer displays alert message
```

> title: UI error when editing smartrule from application context v2 (#5430) [GRAFT][release/cd] (#5646)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix UI Error when Editing Smart Rule from Application Context
```

> title: UI error when editing smartrule from application context v2 (#5430) [GRAFT][release/cd] (#5646)

Reviewer 2: ```suggestion
title: Fix UI error when editing Smart rule from Application context
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-53381
version: 1019.6.11
---

> UI error when editing smartrule from application context v2 (#5430) [GRAFT][release/cd] (#5646)

Reviewer 2: ```suggestion
Fixed an UI error that occurred when editing a Smart rule from the Application context. This prevents sending request with wrong MO id and then displaying alert message in UI when making edits to Smart rules.
```

> UI error when editing smartrule from application context v2 (#5430) [GRAFT][release/cd] (#5646)

Author: The Claude AI suggested to use the following body:
```suggestion
Fixed a UI error that occurred when editing a smart rule from the application context version 2. This prevents the UI from crashing when making edits to smart rules.
```

Reviewer 2: Just as above. Change this suggestion a little.

> Fixed an UI error that occurred when editing a Smart rule from the Application context. This prevents sending request with wrong MO id and then displaying alert message in UI when making edits to Smart rules.

Reviewer: ```suggestion
Fixed an UI error that occurred when editing a smart rule from the application context. This fix prevents sending a request with a wrong managed object ID and then displaying an alert message.
```
